### DISCUSSION TITLE
Menu not usable / hardware issue (debounce?)

### DISCUSSION THREAD

**Discussion by jueank** [deividAlfa/stm32_soldering_iron_controller#127] [General]
Hey!
with the newest version of deividAlfas firmware, I notice that the menu navigation is unusable (for me & my device). I cannot enter editing values, I cannot go to sleep mode, etc.
I have a KSGER with pcb V2.
While trying to sort out issues on my side, checking again which pcb version I have. I saw that there seems to be a capacitor missing on my pcb. See C4 on the pcb.
Can it be that the presence of this capacitor has an influence of the debouncing of the rotary encoder / button?
Can someone recommend, which capacitor I should solder in as a replacement?

This is a photo of my personal KSGER pcb. The solder point look like there was a capacitor, but it fell off because of bad soldering.
![20240419_101632](https://github.com/deividAlfa/stm32_soldering_iron_controller/assets/42673804/2918d8db-f18b-40e8-936b-efddc5d41752)

The following is the official photo of a KSGER v2 pcb from this github repository in the documentation. C4 is present there:
https://raw.githubusercontent.com/deividAlfa/stm32_soldering_iron_controller/master/Readme_files/boards/ksger_v2_7.jpg

@deividAlfa Maybe you know..?

**Comment by jueank** [comment]
Soldered in a 100nF capacitor. Rotary Encoder + Button works flawless now! Menus + UI are editable as expected!

**Comment by deividAlfa** [comment]
Then your encoder is broken!
